numba/tests/doc_examples/test_interval_example.py

```diff
@@ -105,22 +105,48 @@ def impl_interval(context, builder, sig, args):
         # magictoken.interval_lower_builtin.end
 
         # magictoken.interval_unbox.begin
+        from numba.core.boxing import early_exit_if, early_exit_if_null
```

Comment:
Are imports from `numba.core` allowed? If not, where should these functions be moved / exported to?

Reply:
Probably they should be equipped with docstrings and re-exported from `numba.extending`. That would be ideal

Reply:
I've momved them to `cgutils`, since they're not unique to boxing/unboxing anyway.